Formula/f/foundry.rb

```diff
@@ -0,0 +1,68 @@
+class Foundry < Formula
+  desc "Blazing fast, portable and modular toolkit for Ethereum application development"
+  homepage "https://github.com/foundry-rs/foundry"
+  url "https://github.com/foundry-rs/foundry/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "6c2e543baadc3bf2cdaf44fe1e916e0c27501a333ddc56f15ee0aea5ace90cb3"
+  license any_of: ["MIT", "Apache-2.0"]
+  head "https://github.com/foundry-rs/foundry.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "help2man" => :build
+  depends_on "rust" => :build
+
+  on_macos do
+    # https://github.com/foundry-rs/foundry/blob/51b75c83045a963a48f9cd8d765e0f2e775acb3e/foundryup/install#L56-L59
+    depends_on "libusb"
+  end
+
+  def binaries
+    %w[forge cast anvil chisel].freeze
+  end
+
+  def install
+    if build.stable?
+      ENV["TAG_NAME"] = "stable"
+      ENV["VERGEN_GIT_SHA"] = tap.user.ljust(40)
+    end
+
+    binaries.each do |binary|
```

Comment:
since binaries only has one use, you can just inline `%w[forge cast anvil chisel]`